src/SearchResults.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable indent*/
+import React, { useState } from "react";
+
+import bookingData from "./data/fakeBookings.json";
+function LengthOfStay(endDate, startDate) {
+  const diff = new Date(endDate) - new Date(startDate);
+  return Math.floor(diff / 86400000);
+}
```

Comment:
Good way of finding the difference between two dates. 👍

Tips for more readable code:
- use `differenceInMilliseconds` instead of `diff`
- define `const millisecondsInOneDay = 24*3600*1000;` and return `Math.floor(differenceInMilliseconds / millisecondsInOneDay`

Reply:
> Good way of finding the difference between two dates. 👍
> 
> Tips for more readable code:
> 
> * use `differenceInMilliseconds` instead of `diff`
> * define `const millisecondsInOneDay = 24*3600*1000;` and return `Math.floor(differenceInMilliseconds / millisecondsInOneDay`

Thanks Philip. I've changed it now.